### ISSUE TITLE
[OUDS] [Boosted] Missing aria-pressed="false" at toggle buttons initialization

### ISSUE THREAD

**Issue by hannahiss** [Orange-OpenSource/Orange-Boosted-Bootstrap#3509]
### Prerequisites

- [x] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [x] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

As indicated in [aria-pressed documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-pressed) and in the [aria toggle button pattern documentation](https://www.w3.org/WAI/ARIA/apg/patterns/button/), a toggle button should always have an `aria-pressed` attribute indicating its toggle nature, even when the value is `false` (because default value is `indeterminate` and not `false`).

In our doc, coming from Bootstrap, at the initialization, `aria-pressed="false"` is never set. 

It should be added, in particular in OUDS "chips" component. 

To see for a Boosted fix, and to report to Bootstrap. 

### Reduced test cases

Vocalization of a show password button without `aria-pressed`: "Show password  bouton"

Vocalization of a show password button with `aria-pressed="false"`: "Show password  bouton bascule  non enfoncé"

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome

### What version of Boosted are you using?

OUDS v1.1
